frontend/app/performance/apps/dashboard_app.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import matplotlib.pyplot as plt
+import plotly.express as px
+import seaborn as sns
+
+import streamlit as st
+
+st.set_page_config(layout="wide", page_title="Dashboard")
+st.title("Dashboard")
+
+
+def load_data():
+    return sns.load_dataset("iris")
```

Comment:
nit: it could make sense to use `@st.cache_data` here with the effect that we have our cache lookup in the measurement (and not seaborn's load_dataset)